.flake8

```diff
@@ -0,0 +1,16 @@
+[flake8]
```

Comment:
I'm not sure but now we are using flake8 through `pysen` so this setting file for `flake8` might be unnecessary.